src/Catty/DataModel/UserDataContainer/UserDataContainer.swift

```diff
@@ -154,7 +154,7 @@
             allVariables = vars
         }
 
-        let allObjects = project.allObjects()
+        let allObjects = project.scene.objects()
```

Comment:
Please replace with the `allObjects` method (it is important that later it contains all objects and not only the objects of one Scene.